tests.yaml

```diff
@@ -626,6 +659,32 @@ tests:
         description: 'The readiness status of node minikube has changed 9 times in the last 15 minutes.'
         runbook_url: 'https://github.com/kubernetes-monitoring/kubernetes-mixin/tree/master/runbook.md#alert-name-kubenodereadinessflapping'
 
+- interval: 1m
+  input_series:
+  # This metric only appears with non-zero values, meaning the transition from 0 --> 1 doesn't trigger an alert
+  # However, since that's undesired behavior it'd be kinda pointless to test for it, even though it's expected.
+  - series: 'kubelet_evictions{eviction_signal="memory.available",endpoint="https-main",cluster="kubernetes",instance="10.0.2.15:10250",job="kube-state-metrics",namespace="monitoring",node="minikube",pod="kube-state-metrics-b894d84cc-d6htw",service="kube-state-metrics"}'
+    values: '1 1 1 1 1 1 2 2 2 2 2 2 2 2 2 3 3 3 3 3'
+  alert_rule_test:
+  - eval_time: 18m
```

Comment:
Based on the `rate(...[15m])`, when's the **earliest** the alert would fire with this series? For example, would it fire at `7m` (or maybe `8m`) when the first transition occurs?

Again, maybe add another earlier eval_time showing the alert not firing.

Reply:
it should fire immediately, as soon as the counter increments.  and it will take 15mins of no-incrementing to close the alert